### ISSUE TITLE
Some devices incapable of capabiltieis requests/responses

### ISSUE THREAD

**Issue by mill1000** [mill1000/midea-ac-py#198]
There's been a recent uptick in user reporting connection issues. Some are related to Maximum Connection Lifetime, but other are actually due to the device not responding to capability requests and/or other commands.

[#188](https://github.com/mill1000/midea-ac-py/issues/188)
https://github.com/mill1000/midea-ac-py/issues/187#issuecomment-2253206067

There must be a family of devices out there with a very limited feature set that don't support capabilities, humidity, power, and properties.

To reduce log and issue noise, we should consider adding an option to enable/disable capability requests and/or determine a method to check capability support without generating excessive log noise.

**Comment by mill1000** [comment]
Saw this again in https://github.com/mill1000/midea-ac-py/issues/201#issuecomment-2255345103

Did I break something? Did Midea update the device firmware?

**Comment by sdelmas** [comment]
Aha... this seems to be the one to watch/help with :-).

Ok, the firmware version on the device that's failing is v.082122, which is the same as the devices that are working fine. As far as I know no firmware update has happened lately (certainly none triggered by me). The NetHome Plus App has no issues with any of the devices. Neither does the "Midea AC Lan" integration have issues with the connection (I have that running in parallel just for testing). All settings are the same between the devices (device side as well as in the integration, I am using 60 seconds timeout, but also tried 90 just in case). 

Initially two of the devices had issues, but one seems to have gone to working, I haven't seen any issues for that device, while the other one is permanently malfunctioning.

My access point to wifi is abou 20-30 feet way and the signal is quite strong. Nothing has changed on the network side either. I haven't been using "Midea AC Smart" long enough to know whether this would ever have worked for my failing device though (I installed the integration last week, and it has always had the issue).

Please let me know if there is anything I can do to help with this. Happy to help...

**Comment by mill1000** [comment]
Thanks for the information about the firmware version.

> Neither does the "Midea AC Lan" integration have issues with the connection (I have that running in parallel just for testing). 

I would recommend against this. My experience has shown the devices do not behave well when multiple integrations are trying to query them simultaneously.

You probably aren't seeing connection errors with `georgezhao2010/midea_ac_lan` because that integration logs all it's network errors at the DEBUG level. That's fine, I just prefer that errors are treated as actual errors instead of silently ignoring them.

**Comment by sdelmas** [comment]
I only re-installed it due to the connection issues described here so I can still control the failing device. It will be removed once this is sorted out 😀.

**Comment by mill1000** [comment]
> It will be removed once this is sorted out 😀.

Sorry I will not provide debug support while `midea_ac_lan` is concurrently installed & enabled. I _know_ that too many connections to the device can cause issues. While it may not be responsible for your particular issue I don't want it there to confound the problem.

You have yet to provide debug logs which also limits my ability to help.

**Comment by mill1000** [comment]
Thank you.

Ok, here's what I can tell from your logs so far. There are 3 devices.
* 192.168.10.27
* 192.168.10.82
* 192.168.10.181

Of these, 192.168.10.181 is the most "normal". It responds to capability requests and didn't have any connection problems.

192.168.10.27 & 192.168.10.82 don't respond to capability requests, but this is a relatively harmless failure.

192.168.10.82 appears to be working just fine. 

192.168.10.27 fails to respond to energy usage requests. This causes the connection to be killed and causes the AC to go unavailable. You should disable the energy and power entities for this device in HA.

**Comment by mill1000** [comment]
@curtovic Please create another issue for your problem. Your logs weren't run long enough with debug logging enabled to see the issue occur.

**Comment by sdelmas** [comment]
Confirmed... those power entities when what caused the device to shut down. Disabled them, and things are working fine. Thank you for digging into this and solving my problem.

**Comment by mill1000** [comment]
Glad we were able to resolve this for you.
